src/lib/types/Augments.d.ts

```diff
@@ -222,6 +222,7 @@ declare module 'discord.js' {
 		queueFn(fn: (user: KlasaUser) => Promise<T>): Promise<T>;
 		bank(options?: GetUserBankOptions): Bank;
 		getPOH(): Promise<PoHTable>;
+		getUserFavAlchs(): Item[];
```

Comment:
if this is going to be an extendable, then it shouldnt have `User` in the name, my suggestion wasnt for an extendable but just a regular function taking the user as an arg.